integrations/appengine/README.md

```diff
@@ -1,4 +1,4 @@
-## Deploy Static Website on App Engine
+## Deploy Static Website on App Engine for The CHIP Coverage Report
```

Comment:
Does the report call itself CHIP coverage? Can this instead be called something like the "Matter SDK Coverage Report"?

Reply:
It didn't actually have any title.
Updated both the readme and the script to "Matter SDK Coverage Report" and added the SHA to the report as well. Now it looks like this:

![image](https://github.com/user-attachments/assets/d2608267-0c8f-4fef-8d21-2f0e11bdff44)